### PR TITLE
Added test for the way BytePairTokenizer handles the \n\n sequence, which is important in Lama chat templates

### PR DESCRIPTION
**Pull request by martin-gorner**
this also removes the @pytest.mark.large from the BytePairTokenizer test which only takes a second to run.

### CODE REVIEW

File: keras_hub/src/tokenizers/byte_pair_tokenizer_test.py [hunk 1]
```diff
@@ -111,6 +111,24 @@ def test_whitespace_split(self):
         encoded = self.tokenizer(input_data)
         self.assertAllEqual(encoded, [1437, 1437, 50140, 50118, 29])
 
+        # This is important for Llama3 which uses the \n\n sequence in chat
+        # templates: \n\n must be tokenized as a single token
+        input_data = "Hello\n\nHello"
+        encoded = self.tokenizer(input_data)
+        # self.assertAllEqual(encoded, [31414, 50140, 31414])
```
Comment (SamanehSaadat) [inline keras_hub/src/tokenizers/byte_pair_tokenizer_test.py:118]: Should this be uncommented?

File: keras_hub/src/tokenizers/byte_pair_tokenizer_test.py [hunk 2]
```diff
@@ -111,6 +111,24 @@ def test_whitespace_split(self):
...
+
+        input_data = "Hello\n\n\n\nHello"
+        encoded = self.tokenizer(input_data)
+        # self.assertAllEqual(encoded, [31414, 50140, 50140, 31414])
```
Comment (SamanehSaadat) [inline keras_hub/src/tokenizers/byte_pair_tokenizer_test.py:122]: same here

File: keras_hub/src/tokenizers/byte_pair_tokenizer_test.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 )
 
 
-@pytest.mark.large
+# @pytest.mark.large
```
Comment (SamanehSaadat) [inline keras_hub/src/tokenizers/byte_pair_tokenizer_test.py:18]: I think it'd be better to remove unused codes so if we don't want to mark this as large anymore, it'd be better to remove the line.

**Review by SamanehSaadat — COMMENTED** [review]
Thanks, Martin!
Left some nit comments!

File: keras_hub/src/tokenizers/byte_pair_tokenizer_test.py (hunk 3 above)
Comment (martin-gorner) [reply keras_hub/src/tokenizers/byte_pair_tokenizer_test.py:18]: ok

File: keras_hub/src/tokenizers/byte_pair_tokenizer_test.py (hunk 1 above)
Comment (martin-gorner) [reply keras_hub/src/tokenizers/byte_pair_tokenizer_test.py]: yes, my bad

File: keras_hub/src/tokenizers/byte_pair_tokenizer_test.py (hunk 3 above)
Comment (martin-gorner) [reply keras_hub/src/tokenizers/byte_pair_tokenizer_test.py]: removed

**Review by SamanehSaadat — APPROVED** [review]
Thanks, Martin!

**Pull request merged**